src/components/SalesChannel.js

```diff
@@ -0,0 +1,321 @@
+                                    {__("Fullfillment By Amazon (FBA)", "wp-module-ecommerce")}
+                                </h2>
+                                <p className="nfd-text[#000000]">
+                                    {__("Auto-export non-Amazon orders to FBA. Import the tracking and pass it to the", "wp-module-ecommerce")}
+                                <br />
+                                {__("originating sales channel, marking it as shipped.", "wp-module-ecommerce")}
+                                </p>
+                            </div>                            
+                        </div>
+
+                        <div className="nfd-flex nfd-flex-wrap nfd-flex-row nfd-min-w-[315px] nfd-max-w-[646px] nfd-mx-auto">
+                            <div className="nfd-flex-1 nfd-mr-14">
+                                <div>
+                                    <div className="nfd-flex nfd-flex-row">
+                                        <Barcodes className="nfd-mr-4" />
+                                        <h3 className="nfd-text-[#111729] nfd-text-base nfd-leading-6 nfd-font-medium">
+                                            {__("Barcodes", "wp-module-ecommerce")}
+                                        </h3>
+                                    </div>
+                                    <div className="nfd-text-[#000000] nfd-leading-5 nfd-font-medium nfd-mt-5 nfd-mb-11">
+                                        {__("Print barcodes for your products that can be", "wp-module-ecommerce")}
+                                        <br />
+                                        {__("used for labedivng and scanning your", "wp-module-ecommerce")}
+                                        <br />
+                                        {__("products", "wp-module-ecommerce")}
+                                    </div>
+                                </div>
+                                <div>
+                                    <div className="nfd-flex nfd-flex-row">
+                                        <ShippingLabels className="nfd-mr-4" />
+                                        <h3 className="nfd-text-[#111729] nfd-text-base nfd-leading-6 nfd-font-medium">
+                                            {__("Shipping Labels", "wp-module-ecommerce")}
+                                        </h3>
+                                    </div>
+                                    <div className="nfd-text-[#000000] nfd-leading-5 nfd-font-medium nfd-mt-5 nfd-mb-11">
+                                        {__("Print shipping labels with a preferred carrier right from your ecomdash account and save on postage.", "wp-module-ecommerce")}
+                                        <br />
+                                        {__("right from your ecomdash account and save", "wp-module-ecommerce")}
+                                        <br />
```

Comment:
these needs to be removed either from the above line or from here 
`right from your ecomdash account and save on postage.